pkg/services/baremetal/host/host.go

```diff
@@ -1202,6 +1205,131 @@ func (s *Service) actionImageInstalling(ctx context.Context) actionResult {
 	}
 }
 
+func (s *Service) actionImageInstallingCustomImageURLCommand(ctx context.Context, sshClient sshclient.Client) actionResult {
+	host := s.scope.HetznerBareMetalHost
+
+	state, logFile, err := sshClient.StateOfImageURLCommand()
+	if err != nil {
+		return actionError{err: fmt.Errorf("StateOfImageURLCommand failed: %w", err)}
+	}
+
+	duration := time.Since(host.Spec.Status.LastUpdated.Time)
+	// Please keep the number (7) in sync with the docstring of ImageURL.
+	if duration > 7*time.Minute {
+		// timeout. Something has failed.
+		msg := fmt.Sprintf("ImageURLCommand timed out after %s. Deleting machine",
+			duration.Round(time.Second).String())
+		s.scope.Logger.Error(nil, msg, "logFile", logFile)
+		conditions.MarkFalse(host, infrav1.ProvisionSucceededCondition,
+			"ImageURLCommandTimedout", clusterv1.ConditionSeverityWarning,
```

Comment:
Timedout -> TimedOut search and replace plz

Reply:
done.